openmrs-watcher/src/test/java/org/openmrs/eip/mysql/watcher/route/DbEventProcessorRouteTest.java

```diff
@@ -215,13 +213,15 @@ public void shouldNotProcessADeletedOrderThatHasAPreviousOrderIfThePreviousOrder
 		assertMessageLogged(Level.DEBUG, END_ROUTE_MSG);
 	}
 	
+	@Disabled
```

Comment:
Can we rather fix than disable which is one of the major motivation for this PR?